lib/timers/promises.js

```diff
@@ -111,7 +115,70 @@ function setImmediate(value, options = {}) {
       () => signal.removeEventListener('abort', oncancel)) : ret;
 }
 
+async function* setInterval(after, value, options = {}) {
+  validateObject(options, 'options');
+  const { signal, ref = true, throwOnAbort = true } = options;
```

Comment:
Note the docs say this (throwOnAbort) defaults to false - I think the option can probably just be removed.

Reply:
Obviously a mistake in the docs. However, It looks like the consensus is to remove the flag anyway.

Reply:
removed throwOnAbort